### PR TITLE
nls: Mark more strings and add translations

### PR DESCRIPTION
**Pull request by andy5995**

### CODE REVIEW

File: po/pt_BR.po [hunk 1]
```diff
@@ -7,9 +7,9 @@ msgid ""
 msgstr ""
 "Project-Id-Version: netpanzer\n"
 "Report-Msgid-Bugs-To: \n"
-"POT-Creation-Date: 2024-01-24 17:57-0600\n"
+"POT-Creation-Date: 2024-01-28 03:36-0600\n"
 "PO-Revision-Date: 2024-01-24 16:12-0600\n"
-"Last-Translator: none <no-one@nowhere.net>\n"
+"Last-Translator: ChatGPT\n"
 "Language: pt_BR\n"
```
Comment (andy5995) [inline po/pt_BR.po:13]: @ph2708 What do you think of ChatGPT's translations?

**Comment by andy5995** [comment]
image

**Comment by andy5995** [comment]
I suppose there's no point to this until the layout is changed to account for the variable width of the translated strings.

**Comment by winrid** [comment]
@andy5995 you can keep this open! I think supporting variable length strings is something to support sooner than later.

We should just continue to improve the existing UI system for now to unblock improvements, until someone has time to support an alternative. I did not find a nice alternative yet.

Supporting variable length strings is relatively trivial in complexity, however it is a lot of grunt work... :) I migrated a lot of the UI already - just have to call the appropriate Surface::getStringWidth/Height when creating your surface. At some point we should have a "TextSurface" that auto sets up its width and supports line wrapping - maybe that is something that can be done sooner or later.

**Comment by andy5995** [comment]
Ok, I'll have to recreate the branch

**Comment by winrid** [comment]
Progress to unblock this here: https://github.com/netpanzer/netpanzer/pull/214

**Comment by winrid** [comment]
@andy5995 variable width wrapped text support for help and credits merged: https://github.com/netpanzer/netpanzer/pull/230

**Comment by andy5995** [comment]
Nice work @winrid ! Unfortunately I have no idea when I'll get back to this. Absolutely no interest, atm, tbh.

**Comment by andy5995** [comment]
Updated and fixed the merge conflicts, ready to review @winrid

**Review by winrid — APPROVED** [review]

**Comment by winrid** [comment]
amazing, thank you!

**Pull request merged**